app/controllers/org_admin/templates_controller.rb

```diff
@@ -151,19 +156,19 @@ def destroy
         if current == @template
           if @template.destroy
             flash[:notice] = success_message(_('template'), _('removed'))
-            redirect_to org_admin_templates_path
+            redirect_to org_admin_templates_path(r: current_tab)
           else
             @hash = @template.to_hash
             flash[:alert] = failed_destroy_error(@template, _('template'))
-            render org_admin_templates_path
+            render org_admin_templates_path(r: current_tab)
```

Comment:
I'm not aware of render expecting a URL as argument. Is this a copy/paste thing?

Reply:
yep good catch. should just be doing a redirect like the others